code/game/gamemodes/endgame/scoreboard/scoreboard.dm

```diff
@@ -87,21 +87,21 @@ var/global/datum/controller/gameticker/scoreboard/score = new()
 	dat += "[ticker.mode.dat]<HR>" //figure this out
 
 	//populate scores
-	dat += medbay_score(dat)
-	dat += engineering_score(dat)
-	dat += service_score(dat)
-	dat += supply_score(dat)
-	dat += science_score(dat)
+	dat = medbay_score(dat)
+	dat = engineering_score(dat)
+	dat = service_score(dat)
+	dat = supply_score(dat)
+	dat = science_score(dat)
 	if(NO)
 		dat += nuke_op_score(NO)
 	if(RV)
 		dat += revolution_score(RV)
 	if(TR)
 		dat += syndicate_score(TR)
-	dat += silicon_score(dat)
-	dat += misc_score(dat)
+	dat += silicon_score()
```

Comment:
Why do some of them add and some them assign?

Reply:
good question, I'm going to make another PR to make everything consistent after this is hotfixed. I don't like how it works currently so I'll change it shortly